src/fund/shares/Shares.sol

```diff
@@ -251,68 +251,54 @@ contract Shares is IShares, TokenUser, Spoke, SharesToken {
     /// @notice Redeem a specified quantity of the sender's shares
     /// for a proportionate slice of the fund's assets
     /// @param _sharesQuantity The amount of shares to redeem
```

Comment:
1. Add `/// @param _bypassFailure True if token transfer failures should be ignored and forfeited`
2. Add `/// @dev If _bypassFailure is set to true, the user will lose their claim to any assets for which the transfer function fails. This should always be false unless explicitly intended`

Reply:
Ok, so since we're changing to the new functions, we don't need the `@param`, but `redeemSharesEmergency` should include the `@dev` comment